index.js

```diff
@@ -1,10 +1,8 @@
-module.exports = SemanticReleaseError;
-
-SemanticReleaseError.prototype = new Error();
-
-function SemanticReleaseError(message, code) {
-  Error.captureStackTrace(this, this.constructor);
-  this.name = this.constructor.name;
-  this.message = message;
-  this.code = code;
-}
+module.exports = class SemanticReleaseError extends Error {
+  constructor(message, code) {
+    super(message);
+    Error.captureStackTrace(this, this.constructor);
+    this.name = this.constructor.name;
```

Comment:
so if I define a custom Error that extends SemanticReleaseError, it will have the name of that class and not `SemanticReleaseError` - is that indented?

Reply:
I think it makes sense this way. When you define custom errors, the name will be set based on their class declaration

```js
const SemanticReleaseError = require('@semantic-release/error')
class MyCustomError extends SemanticReleaseError {}
console.log((new MyCustomError).name)
// logs "MyCustomError"
```

see https://runkit.com/59f27831133fc9001297f7c0/59f60a7041b72a001258c85c